shared/todo.test.js

```diff
@@ -13,56 +12,67 @@ beforeEach(() => {
   deleteFile(`${__dirname}/done.txt`);
 });
 
-let todoTxtCli = (...args) => [`${__dirname}/todo.sh`, ...args].join(" ");
+let executable = process.platform === "win32" ? "todo.bat" : "todo.sh";
+
+let todoTxtCli = (...args) => [`${__dirname}/${executable}`, ...args].join(" ");
 
 let usage = `Usage :-
-$ ./todo.sh add "todo item"  # Add a new todo
-$ ./todo.sh ls               # Show remaining todos
-$ ./todo.sh del NUMBER       # Delete a todo
-$ ./todo.sh done NUMBER      # Complete a todo
-$ ./todo.sh help             # Show usage
-$ ./todo.sh report           # Statistics
-`;
+$ ./todo.exe add "todo item"  # Add a new todo
+$ ./todo.exe ls               # Show remaining todos
+$ ./todo.exe del NUMBER       # Delete a todo
+$ ./todo.exe done NUMBER      # Complete a todo
+$ ./todo.exe help             # Show usage
+$ ./todo.exe report           # Statistics`;
 
 test("prints help when no additional args are provided", () => {
   let received = execSync(todoTxtCli()).toString("utf8");
 
-  expect(received).toBe(usage);
+  expect(received).toEqual(expect.stringContaining(usage));
 });
 
 test("prints help", () => {
   let received = execSync(todoTxtCli("help")).toString("utf8");
 
-  expect(received).toBe(usage);
+  expect(received).toEqual(expect.stringContaining(usage));
 });
 
 test("add a single todo", () => {
-  let expected = `Added todo: "the thing i need to do"${EOL}`;
-  let received = execSync(todoTxtCli("add", '"the thing i need to do"')).toString("utf8");
+  let expected = 'Added todo: "the thing i need to do"';
+  let received = execSync(
+    todoTxtCli("add", '"the thing i need to do"')
+  ).toString("utf8");
 
-  expect(received).toBe(expected);
+  expect(received).toEqual(expect.stringContaining(expected));
 });
 
 test("show error message when add is not followed by a todo", () => {
-  let expected = "Error: Missing todo string. Nothing added!" + EOL;
```

Comment:
I didn't get any test failures on either Ubuntu or Windows. 🤔 

Also, why would it fail? I've updated all tests to check for the _inclusion_ of the minimum required string, and not to look for an exact match.

I did this because console output on Windows & *nix were not exactly identical in all circumstances - mainly the addition or absence of _EOL_-s. So I got rid of manually added _EOL_-s from check-strings entirely.

Reply:
Ok I missed the change of matcher to `expect.stringContaining`. This looks good then.